transcom/mymove
# INT B-20618 office edit mobile home hotfix

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20618)

## Original INT PR
#13650 

## Rebase INT PR
#13930 

## Summary

This branch fixes an issue where the header for the shipment card was not visible for Mobile Home shipmens both in the approval preview window when a TOO would click "Approve Selected", and also on the "Move Task Order" tab after the shipment was approved.

### How to test

1. Access a move as a TOO, either one with a Mobile Home shipment or one without, you can just add one with the "Add shipment" button.
2. Check the shipment and click "Approve selected". Confirm that the "Mobile Home" title is visible on the shipment card inside the pop-up modal.
3. Once approved, go to the "Move Task Order" tab and confirm that the shipment is visible, and the title is visible at the top of the card.
